File: src/plugin/npc/ThessaliaNPCPlugin.java
package plugin.npc;

import org.redrune.game.content.PlayerLook;
import org.redrune.game.entity.actor.npc.NPC;
import org.redrune.game.entity.actor.player.Player;
import org.redrune.game.plugin.type.NPCPlugin;

/**
 * @author Tyluur <itstyluur@gmail.com>
 * @since 8/31/2017
 */
public class ThessaliaNPCPlugin extends NPCPlugin {
	
	@Override
	public boolean handle(Player player, NPC npc, String option) {
		switch (option) {
			case "Change-clothes":
				PlayerLook.openThessaliasMakeOver(player);
				return true;
		}
		return false;
	}
	
	@Override
	public void register() {
		register(548, "Change-clothes");
	}
}

File: src/main/kotlin/plugin/npc/ThessaliaNPCPlugin.kt
package plugin.npc

import org.redrune.game.content.entity.actor.player.PlayerLook
import org.redrune.game.content.plugin.type.NPCPlugin
import org.redrune.game.entity.actor.npc.NPC
import org.redrune.game.entity.actor.player.Player

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 8/31/2017
 */
class ThessaliaNPCPlugin : NPCPlugin {

    override fun handle(player: Player, npc: NPC, option: String): Boolean {
        when (option) {
            "Change-clothes" -> {
                PlayerLook.openThessaliasMakeOver(player)
                return true
            }
        }
        return false
    }

    override fun register() {
        registerNPC(548, "Change-clothes")
    }
}